src/__test__/Results.test.tsx

```diff
@@ -2,30 +2,127 @@ import Results from "../components/Results";
 import React from "react";
 import ReactDOM from "react-dom";
 import { render, cleanup } from "@testing-library/react";
+import { CATEGORY_NAMES } from "../constants";
+
+const defaultAllCorrectAnswers = {
+  accessibilityQuestionsCorrect: 3,
+  agileQuestionsCorrect: 3,
+  cssQuestionsCorrect: 0,
+  generalCSQuestionsCorrect: 0,
+  gitQuestionsCorrect: 0,
+  htmlQuestionsCorrect: 0,
+  infoTechQuestionsCorrect: 0,
+  javascriptQuestionsCorrect: 0,
+  linuxQuestionsCorrect: 2,
+  pythonQuestionsCorrect: 2,
+  qualityAssuranceQuestionsCorrect: 0,
+  securityQuestionsCorrect: 0,
+  sqlQuestionsCorrect: 0
+};
+
+const allWrongAnswers = {
```

Comment:
We need to add `freeCodeCampCorrect: 0,` because that category is listed in the CATEGORY_NAMES constant